schemas/mw-page-delete.yaml

```diff
@@ -0,0 +1,53 @@
+title: MediaWiki Page Delete
+description: Represents a MW Page Delete event
+type: object
+properties:
+  # global event fields
+  meta:
```

Comment:
Last minute alternative idea: `envelope`, as in the letter envelope. Slightly clearer and more specific, imho.